lib/v2/bilibili/cache.js

```diff
@@ -1,27 +1,75 @@
 const got = require('@/utils/got');
+const UA = require('@/utils/rand-user-agent')({ browser: 'chrome', os: 'windows', device: 'desktop' });
```

Comment:
Does UA really matter since RSSHub is using `{ browser: 'chrome', os: 'mac os', device: 'desktop' }` by default